hudi-common/src/main/java/org/apache/hudi/common/table/timeline/HoodieArchivedTimeline.java

```diff
@@ -84,6 +90,38 @@ public boolean isInRange(String instantTime) {
     }
   }
 
+  class InclusiveStartAndEndTsFilter extends TimeRangeFilter {
```

Comment:
Let's name it as `ClosedClosedTimeRangeFilter`.

Reply:
Fixed.